templates/shared/searchfuzzy.tmpl

```diff
@@ -0,0 +1,13 @@
+<!-- Topic can be "repo" or "explore" -->
```

Comment:
Can we perhaps document **all** parameters needed for shared templates at the top?